### PR TITLE
UI/Storybook

### PR DESCRIPTION
**Pull request by borut-t**
Created a storybook where all our UI components will reside.

Simulator Screen Shot - iPhone 14 Pro - 2023-02-13 at 15 16 25 2

### CODE REVIEW

File: Demo/Storybook/Storybook/Component.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//
+//  Component.swift
+//  Storybook
+//
+//  Created by Borut Tomazin on 23/01/2023.
+//
+
+import Foundation
+
+enum Component: String, Identifiable, Hashable, CaseIterable {
```
Comment (gentique) [inline Demo/Storybook/Storybook/Component.swift:10]: Based on the usage we can omit conformance to all except `CaseIterable`.

on `ContentView` we can change the ForEach method to the following 
```swift
ForEach(Component.allCases, id: \.self)
```

**Review by gentique — APPROVED** [review]
Except for the minor nitpick, this looks really good! I like the direction this is taking 🚀

File: Demo/Storybook/Storybook/Component.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  case actionButton
+  case paddingLabel
+  
+  var id: RawValue { rawValue }
```
Comment (gentique) [inline Demo/Storybook/Storybook/Component.swift:14]: This can also be omitted.

File: Demo/Storybook/Storybook/Component.swift (hunk 1 above)
Comment (borut-t) [reply Demo/Storybook/Storybook/Component.swift:10]: True. Thanks 🙌

**Review by markom-povio — APPROVED** [review]
Nice 👍

**Pull request merged**